static/app/components/events/autofix/autofixSolution.tsx

```diff
@@ -514,7 +516,11 @@ function AutofixSolutionDisplay({
                       ? t(
                           "Seer can't access any of your selected repos. Check your GitHub integration and make sure Seer has read access."
                         )
-                      : undefined
+                      : codingDisabled
```

Comment:
should this be first match in the ternary conditions? some org admin could disable org-wide and someone in it could be confused trying to fix repo issues before getting to this message